packages/pill/index.js

```diff
@@ -0,0 +1,107 @@
+import "@warp-ds/icons/elements/close-16";
+import { activateI18n } from "../i18n";
+import { classNames } from "@chbphone55/classnames";
+import { html } from "lit";
+import { i18n } from "@lingui/core";
+import { kebabCaseAttributes } from '../utils';
+import { messages as enMessages } from "./locales/en/messages.mjs";
+import { messages as fiMessages } from "./locales/fi/messages.mjs";
+import { messages as nbMessages } from "./locales/nb/messages.mjs";
+import { pill as ccPill } from "@warp-ds/css/component-classes";
+import WarpElement from "@warp-ds/elements-core";
+
+class WarpPill extends kebabCaseAttributes(WarpElement) {
+  static styles = [WarpElement.styles];
+
+  static properties = {
+    label: { type: String },
+    canClose: { type: Boolean },
+    suggestion: { type: Boolean },
+    openSRLabel: { type: String },
+    closeSRLabel: { type: String },
+  };
+
+  constructor() {
+    super();
+    activateI18n(enMessages, nbMessages, fiMessages);
+    this.canClose = false;
+    this.suggestion = false;
+
```

Comment:
Should we also initiate `this.label` with `""` here?

Reply:
I think of `label` as a mandatory field here, hence not setting anything